<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticsPreferencesPopup</class>
 <widget class="QWidget" name="StatisticsPreferencesPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>208</width>
    <height>204</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="check_words">
     <property name="text">
      <string>Words</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="check_sentences">
     <property name="text">
      <string>Sentences</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="check_syllables">
     <property name="text">
      <string>Syllables</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="check_lines">
     <property name="text">
      <string>Lines</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="check_charspace">
     <property name="text">
      <string>Characters With Spaces</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="check_charnospace">
     <property name="text">
      <string>Characters without spaces</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="check_east">
     <property name="text">
      <string>East Asian characters</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="check_flesch">
     <property name="text">
      <string>Flesch Reading Ease</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
